psalm-baseline.xml

```diff
@@ -30,6 +30,11 @@
       <code>$matches[3]</code>
     </PossiblyUndefinedIntArrayOffset>
   </file>
+  <file src="src/Psalm/Internal/Analyzer/AlgebraAnalyzer.php">
```

Comment:
It seems like these new things are because we're replacing `count($foo->clauses)` with `$foo->count()`, and I wonder whether the former is better? It's a bit more verbose, but it allows the typechecker to do its thing.

Reply:
Yeah makes sense, reverted.